src/Lib/Element.php

```diff
@@ -49,6 +60,7 @@ public function getElementName()
      */
     final protected function addOption($key, $value, $isInternal = true)
     {
+        $key = (string) $key;
```

Comment:
Fields in elasticsearch can be named anything. If a field name is an integer which is out of bounds for any reason, the options array would be corrupt, unless it is first cast to a string.